src/_guides/libraries/library-tour.md

```diff
@@ -1111,24 +1102,39 @@ In the preceding example, the methods run in the following order:
 2.  `expensiveWork()`
 3.  `lengthyComputation()`
 
+Here is the same code written using await:
+
+<?code-excerpt "lib/library_tour/async/future.dart (then-chain-as-await)"?>
+{% prettify dart %}
+try {
+  final value = await costlyQuery(url);
+  await expensiveWork(value);
+  await lengthyComputation();
+  print('Done!');
+} catch (e) {
+  /* Handle exception... */
+}
+{% endprettify %}
+
```

Comment:
Given that the subsection before this one already covers "Using await", it seemed to make sense to include a version of the example code that uses await rather than `then()` chaining

Reply:
Just bookmarking in case you do decide to go with my suggestion to move "Basic usage" back to where it was. Not sure if it changes anything about the reasoning here. (I think it doesn't.)

Reply:
> Not sure if it changes anything about the reasoning here. (I think it doesn't.)

Agreed. Leaving this as is.